packages/typespec-go/test/azure-http-specs/azure/client-generator-core/clientinitializationgroup/zz_servicemultipleparams_client.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) Go Code Generator. DO NOT EDIT.
+
+package clientinitializationgroup
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"net/http"
+)
+
+// ServiceMultipleParamsClient contains the methods for the ServiceMultipleParams group.
+// Don't use this type directly, use [ServiceClient.NewServiceMultipleParamsClient] instead.
+type ServiceMultipleParamsClient struct {
+	internal *azcore.Client
+}
+
+// WithBody -
+// If the operation fails it returns an *azcore.ResponseError type.
+//   - options - ServiceMultipleParamsClientWithBodyOptions contains the optional parameters for the ServiceMultipleParamsClient.WithBody
+//     method.
+func (client *ServiceMultipleParamsClient) WithBody(ctx context.Context, name string, region string, body Input, options *ServiceMultipleParamsClientWithBodyOptions) (ServiceMultipleParamsClientWithBodyResponse, error) {
```

Comment:
`region` is supposed to be a client param.

Please remove `clientinitializationgroup`.